tests/phpunit/integration/Modules/Analytics_4Test.php

```diff
@@ -2851,6 +2851,11 @@ public function test_get_custom_dimensions_data() {
 		$data     = $method->invoke( $this->analytics );
 		$this->assertEmpty( $data );
 
+		// Ensure the `'googlesitekit_post_categories'` key is not present in
```

Comment:
Minor typo, I'll fix it now.
```suggestion
		// Ensure the `'googlesitekit_post_categories'` key is not present
```